tests/app/data_model/test_answer_store.py

```diff
@@ -28,6 +28,20 @@ def basic_answer_store():
     answer_store.add_or_update(Answer(answer_id="answer3", value=30))
     answer_store.add_or_update(Answer(answer_id="another-answer3", value=35))
 
+    answer_store.add_or_update(
+        Answer(answer_id="answer4", value="<p>abc123</p>", list_item_id="xyz987")
+    )
+    answer_store.add_or_update(
+        Answer(answer_id="answer5", value=["<p>abc123</p>", 1], list_item_id="xyz987")
+    )
+    answer_store.add_or_update(
+        Answer(
+            answer_id="answer6",
+            value={"item1": "<p>abc123</p>", "item2": 1},
+            list_item_id="xyz987",
+        )
+    )
```

Comment:
If `list_item_id` is not a required param for `get_escaped_answer_value`, it can be dropped from the tests.

Reply:
Done, thanks!